packages/canary-web-components/src/components/ic-menu-with-multi/ic-menu.css

```diff
@@ -0,0 +1,257 @@
+}
+
+:host(.full-width) {
+  width: 100%;
+}
+
+.option {
+  padding: var(--ic-space-xs) 0.438rem;
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+}
+
+.loading-option p {
+  margin-bottom: 0 !important;
+}
+
+.option:not(.loading-option) {
+  justify-content: space-between;
+}
+
+:host(.small) .option {
+  padding: var(--ic-space-xxs) 0.438rem;
+}
+
+:host(.large) .option {
+  padding: var(--ic-space-sm) 0.438rem;
+}
+
+.option:last-child {
+  border-radius: 0 0 1px 1px;
+}
+
+:not(.menu:has(+ .option-bar)) .option:first-child {
+  border-radius: 1px 1px 0 0;
+}
+
+.option:not(.disabled-option):hover {
+  background-color: var(--ic-action-dark-bg-hover);
+}
+
+.option:not(.disabled-option):active {
+  background-color: var(--ic-action-dark-bg-active);
+}
+
+.option:focus-visible {
+  outline: var(--ic-hc-focus-outline);
+}
+
+.option:not(.disabled-option) .option-description {
+  color: var(--ic-color-secondary-text);
+}
+
+.option .option-label {
+  display: flex;
+  flex-direction: row;
+  align-items: top;
+}
+
+.option .option-element {
+  margin-top: var(--ic-space-xxs);
+}
+
+.option .option-icon {
+  height: var(--ic-space-md);
+  width: var(--ic-space-md);
+  margin-top: var(--ic-space-xxs);
+  margin-right: var(--ic-space-xxxs);
+}
+
+.option-text-container {
+  pointer-events: none;
+  max-width: calc(var(--ic-space-lg) * 10 + var(--ic-space-xl));
+}
+
+.option.loading-option .option-text-container {
+  margin-left: 0.625rem;
+  color: var(--ic-color-secondary-text);
+}
+
+.error-icon-svg {
+  height: var(--ic-space-md);
+  width: var(--ic-space-md);
+  fill: var(--ic-status-error);
+  align-self: center;
+  display: flex;
+  margin-right: calc(var(--ic-space-xxs) * 1.2);
+}
+
+.loading-error-info {
+  display: flex;
+}
+
+.check-icon {
+  height: var(--ic-space-lg);
+  width: var(--ic-space-lg);
+  margin-left: var(--ic-space-xs);
+  pointer-events: none;
+}
+
+.focused-option .check-icon * {
+  fill: currentcolor;
+}
+
+.option-group-title {
+  padding: var(--ic-space-lg) 0.438rem var(--ic-space-xs);
+  color: var(--ic-color-tertiary-text);
+}
+
+:host(.small) .option-group-title {
+  padding: var(--ic-space-sm) 0.438rem var(--ic-space-xxs);
+}
+
+.last-recommended-option {
+  border-bottom: var(--ic-border-default);
+}
+
+.disabled-option {
+  color: var(--ic-architectural-200);
+  cursor: default;
+  pointer-events: none;
+}
+
+.focused-option,
+.focused-option .option-description {
+  background-color: var(--ic-focus-blue) !important;
+  color: var(--ic-color-white-text) !important;
+}
+
+:host(.no-results) li {
+  cursor: no-drop;
+}
+
+:host(.no-results) li:hover {
+  background-color: transparent;
+}
+
+.option-bar {
+  padding: var(--ic-space-xxs) var(--ic-space-xs);
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: var(--ic-architectural-40);
+  visibility: hidden;
+  border-bottom: 0.0625rem solid var(--ic-architectural-400);
+  border-radius: 1px 1px 0 0;
```

Comment:
I felt that it was best not to use either of those because it doesn't really fit into either category, i.e. it's not the width of a border and it's not spacing (like padding or a margin), and it's quite a unique value for a border radius because most components have it as 2px so didn't feel it was worth creating a new variable or anything.

I think we want to be strict with only applying those variables to the appropriate CSS property - for example, if we wanted to update all our border widths in future, we wouldn't want other styling to be affected that is not related.

Reply:
For line 233, I felt that it was best not to use either of those because it doesn't really fit into either category, i.e. it's not the width of a border and it's not spacing (like padding or a margin), and it's quite a unique value for a border radius because most components have it as 2px so didn't feel it was worth creating a new variable or anything.

However, I've updated line 232 to use the spacing variable, and also found a few other values in the file which I felt could probably use the variables as well so have updated those too :)